improved_article.py

```diff
@@ -0,0 +1,22 @@
+"""
+Improved article implementation
+"""
+
+import json
+
+from core_utils.article import ASSETS_PATH, Article, ArtifactType
+
+
+class ImprovedArticle(Article):
```

Comment:
you have to implement your pipeline without any modifications of article and introducing new files